configure.ac

```diff
@@ -77,6 +77,27 @@ AC_CHECK_LIB([ibverbs], [ibv_get_device_list],
 AC_CHECK_DECLS(IBV_ACCESS_RELAXED_ORDERING, [], [],
                     [[#include <infiniband/verbs.h>]])
 
+#check for UCX
+AC_ARG_WITH([ucx],
+        AC_HELP_STRING([--with-ucx=PATH], [Path to UCX installation]),
+        [AS_IF([test "x$with_ucx" != xno],
+          [AC_SUBST([UCX_PATH], $with_ucx)
```

Comment:
may be users might just compile w/o any path as UCX is shipped with Linux distros. ucx libs are in /usr/lib64/libucp.so.0

Reply:
need to support that as well